### ISSUE TITLE
поле "Фискальный признак"

### ISSUE THREAD

**Issue by Dafafgh** [VitalyKravtsov2016/OposWebkassa#35]
вопрос по заполнению поля в печатной форме чека "Фискальный признак"(рис.). 
сейчас в нем печатается внутренний номер транзакции АСУ "ExternalCheckNumber":"29218", но это наверное неверно, т.к. фискальный признак это "CheckNumber":"1151393645065"

можно выводить "CheckNumber" в строку чека "Фискальный признак", в "Чек №" убрать из чека?(выделено красным)

[SHTRIH-M-OPOS-1_2022.12.09.log](https://github.com/VitalyKravtsov2016/OposWebkassa/files/10195050/SHTRIH-M-OPOS-1_2022.12.09.log)
![IMG_20221209_160455](https://user-images.githubusercontent.com/112083079/206711328-29e281e0-e7b2-4510-9229-cabc42b17e5a.jpg)

**Comment by Dafafgh** [comment]
можно "CheckNumber" выводить в это место чека?


![IMG_20221213_105834](https://user-images.githubusercontent.com/112083079/207264323-206c7e18-c275-490c-8a9c-ec6d7b11403f.jpg)

**Comment by VitalyKravtsov2016** [comment]
У нас определены команды directIO
  DIO_SET_DRIVER_PARAMETER        = 30; // write internal driver parameter
  DriverParameterExternalCheckNumber = 300;
  DriverParameterFiscalSign          = 301;
  
[13.12.2022 13:55:57.252 ] [00006368] [DEBUG] ToleFiscalPrinter.DirectIO(30, 300, '29242')
[13.12.2022 13:55:57.252 ] [00006368] [DEBUG] ToleFiscalPrinter.DirectIO(30, 300, '29242')=0
[13.12.2022 13:55:57.252 ] [00006368] [DEBUG] ToleFiscalPrinter.DirectIO(30, 301, '29242')
[13.12.2022 13:55:57.252 ] [00006368] [DEBUG] ToleFiscalPrinter.DirectIO(30, 301, '29242')=0

Приложение устанавливает фискальный признак для печати на чеке. 
Зачем тогда это было сделано?

**Comment by Dafafgh** [comment]
так, получается если АСУ не передаст DriverParameterFiscalSign = 301, то в выделенной строке пропишется "CheckNumber"?

если так, то нужно менять в АСУ, не обратил внимания что передаются оба параметра 300, 301

**Comment by VitalyKravtsov2016** [comment]
Сейчас сделано так:  
  if Receipt.FiscalSign = '' then
  begin
    Receipt.FiscalSign := Command.Data.CheckNumber;
  end;
  Document.Add('Фискальный признак: ' + Receipt.FiscalSign);
Если приложение не задало FiscalSign, то используется CheckNumber. 
Напишите как должно работать - сделаем.

**Comment by Dafafgh** [comment]
в приложении отключим передачу FiscalSign
